apps/nextjs-website/src/app/page.tsx

```diff
@@ -41,12 +39,11 @@ const Home = async () => {
       <NotSsrWebinarHeaderBanner webinars={webinars} />
 
       <HeroSwiper
-        cards={homepage.hero.map((itemProp, index) => ({
+        cards={homepage.hero.map((itemProp) => ({
           ...itemProp,
-          child:
-            index === 0 ? (
-              <SiteLabel title={header.title} boldTitle={header.boldTitle} />
-            ) : undefined,
+          child: itemProp.subhead && (
+            <BlocksRendererClient content={itemProp.subhead} />
```

Comment:
As a consequence, this line would become
```suggestion
            <BlocksRendererClient key={index} content={itemProp.subhead} />
```